daemon/algod/api/server/v2/utils.go

```diff
@@ -90,6 +91,13 @@ func byteOrNil(data []byte) *[]byte {
 	return &data
 }
 
+func boolOrNil(b bool) *bool {
```

Comment:
should this be `trueOrNil`?

Reply:
Yes, I think that's a bit more informative